src/util/guard/is-well-formed.test.ts

```diff
@@ -0,0 +1,71 @@
+import {describe, expect, test} from 'vitest';
+
+import {isWellFormed} from '../is-well-formed.js';
+
+describe('true cases', () => {
```

Comment:
as far as I know best practice is to create a describe with the function name and then put the rest of the describes inside that